### ISSUE TITLE
BMiner dont work on different nVidia cards

### ISSUE THREAD

**Issue by Maui73** [nicehash/NiceHashMiner#1769]
The BMiner Plugin don't work on every nVidia cards i have (1050 Ti, 1060, 1070, 1660 Ti, P-106-100 on 2 different systems). The problem exists at any versions of NHM.

Thanx, Maui

[log.txt](https://github.com/nicehash/NiceHashMiner/files/3557217/log.txt)

**Comment by peteycrack420** [comment]
mine doesnt either   after the 10th  are we gonna only have 1 version of nicehash

**Comment by Maui73** [comment]
I have the problem in the current version (1.9.2.13) ... since some versions.

Thanx, Maui

**Comment by AtlAntA118** [comment]
Same problem here. Benchmarking fails.

**Comment by OlivFreitas** [comment]
Same problem... 2 gtx1080 ti  1060 , 1660ti.. Fails  (version 19.2.14)

**Comment by OlivFreitas** [comment]
I managed to solve Win 10 restart issues .. and also the unrecognition of my Gtx1660ti

1st: I disconnected all nvidia GPus and drivers.
2nd: After updating my win10 and finally, I installed 1 GPU with last driver and then installed the other GPUs.

In my case the Win10 restart error randomly occurred while mining.

Hope this helps...
My OS: win10 - 64 v: 1903 with 2-1080ti, 1-1080, 1-1660 ti  NHM-19.2.14

**Comment by OlivFreitas** [comment]
I managed to solve the Win 10 reboot issues .. and also the recognition of my Gtx1660ti

1: I disconnected all Gpu´s and drivers from nvidia.
2nd: After upgrading my win10 and finally I installed 1 GPU with the latest driver and installed the other GPUs.

In my case, the Win10 reboot error occurred randomly during mining.

Now it is normally mining for 3 days !!...

I hope this helps ...
My System: win10 - 64v: 1903 with 2-1080ti, 1-1080, 1-1660 ti  - nhm 19.2.14
